install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ $(whoami) != "root" ]]; then
+  echo "This file needs to be executed as root or with sudo."
+  exit
+fi
+
+defaultdir="/usr/bin"
+
+echo "Directory to install to (default: $defaultdir)"
+read installdir
+
+if [[ $installdir = "" ]]; then
+  installdir=$defaultdir
+fi
+
+if [[ ! -d $installdir ]]; then
+  echo "This directory does not exist."
+  exit
+fi
+
```

Comment:
I think here would be a good place to confirm the place it's going to be installed, by telling the user where it's going to install to and then they say yes or no.

Reply:
```suggestion
echo "Please confirm that you want pridefetch installed to $installdir/pridefetch. This is your last chance to correct any mistakes before we do the install" 
read -p "Continue (y/n)?: " continue
if [ "$continue" != "y" ]; then
  exit 1
fi
```